tests/countries/test_bhutan.py

```
@@ -0,0 +1,152 @@
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.bhutan import Bhutan, BT, BTN
+from tests.common import CommonCountryTests
+
+
+class TestBhutan(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Bhutan, years=(range(1970, 2050)))
```

Pylint (3.3.7)

[convention] Unnecessary parens after '=' keyword

(C0325)